app/Http/Controllers/Admin/HomeworkController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Entry;
+
+class HomeworkController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $homeworks = Entry::where('activity_id', 1)->paginate(15);
```

Comment:
Hoe kan ik dat dan het beste oplossen? Gewoon zorgen dat ID 1 niet verwijderd kan worden? Want er moet maar 1 hw activiteit komen dus om die te achterhalen is wel t handigst met ID 1.